client/src/components/LoginForm/LoginForm.scss

```diff
@@ -40,3 +40,16 @@ h1 {
     color: $primaryColorHover;
   }
 }
+
+.buttonError {
+  @extend %buttonErr;
+  @extend %field;
```

Comment:
`%field` was removed as part of this PR: https://github.com/CodeForSocialGood/calltocode.org/commit/0a4b0026bf932ce78fd20e35cfa7b8733f7e9505